packaging/windows/variables.wxi

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0"?>
 <Include xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <?if $(var.ReleaseSuffix) ~= "beta" OR $(var.ReleaseSuffix) ~= "rc1" OR $(var.ReleaseSuffix) ~= "rtm" OR $(var.ReleaseSuffix) ~= ""?>
+  <?define BuildType="Release"?>
```

Comment:
I can move this logic to the generatemsi.ps1 script and pass the `BuildType` as a variable to the wix files.